### PR TITLE
ED-2386 store retrieve triage answers

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: exportreadiness/models.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from api.model_utils import TimeStampedModel
+
+
+class TriageResult(TimeStampedModel):
+    sector = models.CharField(max_length=255)
+    exported_before = models.BooleanField()
+    exporting_regular_part = models.BooleanField()
+    company_name = models.CharField(max_length=100)
+    sole_trader = models.BooleanField()
+    sso_id = models.PositiveIntegerField(unique=True)
```
Comment (richtier) [inline exportreadiness/models.py]: this could be an FK to supplier. @konradko you dislike FKs, yes?

Comment (konradko) [reply exportreadiness/models.py]: I love FKs when we have something to fk to :P Which is not the case here - user model is in another service

File: exportreadiness/serializers.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+from .models import TriageResult
+
+
+class TriageResultSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = TriageResult
+        extra_kwargs = {
+            'sso_id': {'required': False}
```
Comment (konradko) [inline exportreadiness/serializers.py]: this is defo required - without sso id we will be saving in session

Comment (konradko) [reply exportreadiness/serializers.py]: also, does it have to be `extra_kwargs`? It is a field on `TriageResult` already

File: exportreadiness/models.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+
+from api.model_utils import TimeStampedModel
+
+
+class TriageResult(TimeStampedModel):
```
Comment (konradko) [inline exportreadiness/models.py]: Needs optional CH ID - should be the same name as in company

File: exportreadiness/models.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    sector = models.CharField(max_length=255)
+    exported_before = models.BooleanField()
+    exporting_regular_part = models.BooleanField()
```
Comment (konradko) [inline exportreadiness/models.py]: Think `regular_exporter` is more accurate

File: exportreadiness/models.py (hunk 3 above)
Comment (konradko) [reply exportreadiness/models.py]: Also add `used_online_marketplaces_before`

File: exportreadiness/models.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    company_name = models.CharField(max_length=100)
```
Comment (konradko) [inline exportreadiness/models.py]: this field is optional

File: exportreadiness/models.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+
+from api.model_utils import TimeStampedModel
+
+
+class TriageResult(TimeStampedModel):
+    sector = models.CharField(max_length=255)
```
Comment (konradko) [inline exportreadiness/models.py]: choices will be added in separate PR I understand

File: exportreadiness/models.py (hunk 5 above)
Comment (konradko) [reply exportreadiness/models.py]: Also, why max length 100? We should give max unless there is a good reason

File: exportreadiness/serializers.py (hunk 2 above)
Comment (sdonk) [reply exportreadiness/serializers.py]: It's marked as not required because we inject it in the `create()` method. If it was required the validation would fail and we would never get to create. The client is not passing the `sso_id` in the post data.

File: exportreadiness/models.py (hunk 3 above)
Comment (sdonk) [reply exportreadiness/models.py]: Where does it say that on the ticket?

File: exportreadiness/models.py (hunk 5 above)
Comment (sdonk) [reply exportreadiness/models.py]: `max_length` is required in Django `CharField` so pick a number. It wasn't specified in the ticket, so I picked what I thought it would be a sensible number,

File: exportreadiness/models.py (hunk 6 above)
Comment (sdonk) [reply exportreadiness/models.py]: That was the plan :)

File: exportreadiness/models.py (hunk 3 above)
Comment (konradko) [reply exportreadiness/models.py]: it says so it the comment above! :-)

File: exportreadiness/models.py (hunk 5 above)
Comment (konradko) [reply exportreadiness/models.py]: max possible is more sensible

File: exportreadiness/serializers.py (hunk 2 above)
Comment (konradko) [reply exportreadiness/serializers.py]: can't we set it before validation?

File: exportreadiness/tests/factories.py [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import factory
+
+from exportreadiness.models import TriageResult
+
+
+class TriageResultFactory(factory.django.DjangoModelFactory):
+
+    sector = 'Foo'
+    exported_before = True
```
Comment (konradko) [inline exportreadiness/tests/factories.py]: feels like we should have a random choice from all valid answers here, maybe worth adding a ticket to improve it later

File: exportreadiness/serializers.py (hunk 2 above)
Comment (sdonk) [reply exportreadiness/serializers.py]: Why is it confusing? Technically speaking it's a piece of data we inject and we shouldn't need to validate, so it doesn't belong to `validate` imho. 
Also It's consistent with what we do in other serializers in the code base

File: exportreadiness/models.py (hunk 3 above)
Comment (sdonk) [reply exportreadiness/models.py]: Let me rephrase that, my comment was a bit rushed :)

Will CH id be needed in the future? If yes, we should add it when we need it (ie when we do the CH lookup)

File: exportreadiness/models.py (hunk 5 above)
Comment (sdonk) [reply exportreadiness/models.py]: I don't understand what you mean with max possible

File: exportreadiness/models.py (hunk 1 above)
Comment (sdonk) [reply exportreadiness/models.py]: I actually agree with @konradko here. Having a FK doesn't seem useful here
